src/enums.h

```diff
@@ -363,4 +363,26 @@ std::array<T, N> vec_to_array(const std::vector<T> &vec)
     return array;
 }
 
+template <typename T>
```

Comment:
```
1) Returns exactly c.begin(), which is typically an iterator to the beginning of the sequence represented by c. If C is a standard Container, this returns C::iterator when c is not const-qualified, and C::const_iterator otherwise.
2) Returns a pointer to the beginning of the array array.
3) Returns exactly std::begin(c), with c always treated as const-qualified. If C is a standard Container, this always returns C::const_iterator.
```